lib/puppet/provider/user/windows_adsi.rb

```diff
@@ -109,11 +109,11 @@ def comment=(value)
   end
 
   def home
-    user['HomeDirectory']
+    nil
```

Comment:
Oh, seems that `:absent` is more idiomatic to report a nonexistent property. I'll update the PR to change this once PR testing finishes.

Reply:
Actually nevermind. Since this is a property we are not supporting, it does not make sense to return a value for it (as otherwise, for example, puppet resource would report a value for it which could make the user think that's something they can manage). My question about `:absent` vs. `nil` still stands though -- which should I return?